pyfr/solvers/baseadvecdiff/system.py

```diff
@@ -15,16 +15,33 @@ def deps(dk, *names): return self._kdeps(k, dk, *names)
         g1 = self.backend.graph()
         g1.add_mpi_reqs(m['scal_fpts_recv'])
 
+        # Perform post-processing of the previous solution stage
+        g1.add_all(k['eles/filter_solution'])
+
         # Interpolate the solution to the flux points
-        g1.add_all(k['eles/disu'])
+        g1.add_all(k['eles/disu'], deps=k['eles/filter_solution'])
 
         # Pack and send these interpolated solutions to our neighbours
         g1.add_all(k['mpiint/scal_fpts_pack'], deps=k['eles/disu'])
         for send, pack in zip(m['scal_fpts_send'], k['mpiint/scal_fpts_pack']):
             g1.add_mpi_req(send, deps=[pack])
 
+        # If entropy filtering, compute entropy bounds
```

Comment:
Misleading comment?  As what follows is an MPI request.

Reply:
Updated